### PR TITLE
added instagram model

### PR DESCRIPTION
**Pull request by tanerozel**

### CODE REVIEW

File: Tinder/Models/Instagram.cs [hunk 1]
```diff
@@ -1,6 +1,31 @@
+using System;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
 namespace Tinder.Models
 {
     public class Instagram
```
Comment (verdie-g) [inline Tinder/Models/Instagram.cs]: This class is used in Profile. Could you confirm that the Instagram object in the recommendations is the same as the one in profile?

**Review by tanerozel — COMMENTED** [review]
image
I reviewed it., it does not have problem

File: Tinder/Models/Instagram.cs (hunk 1 above)
Comment (Psoffie009) [reply Tinder/Models/Instagram.cs:7]: > This class is used in Profile. Could you confirm that the Instagram object in the recommendations is the same as the one in profile?

**Comment by Psoffie009** [comment]
> image
> I reviewed it., it does not have problem

**Pull request merged**